lib/rubocop/cop/style/date_time.rb

```diff
@@ -22,8 +22,22 @@ module Style
       #
       #   # good - uses `DateTime` with start argument for historical date
       #   DateTime.iso8601('1751-04-23', Date::ENGLAND)
+      #
+      # @example AllowCoercion: false (default)
+      #
+      #   # bad - coerces to `DateTime`
+      #   something.to_datetime
+      #
+      #   # good - coerces to `Time`
+      #   something.to_time
+      #
+      # @example AllowCoercion: true
+      #
+      #   # good
+      #   something.to_datetime
```

Comment:
Perhaps add that `something.to_time` is also considered `# good` here too?

Reply:
Fixed.